CHANGELOG.md

```diff
@@ -9,6 +9,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Changed: Wording on the Door Locks preset tab to be clearer.
 
+### Metroid: Samus Returns
+
+- Changed: The hinted item after collecting the last required DNA is now dependent on the final boss. The Baby Metroid is hinted for Proteus Ridley, the Ice Beam for Metroid Queen and the Bomb for Diggernaut.
+- Changed: Removed the first Power Beam block before entering the Gamma Arena from Area 4 Crystal Mines - Gawron Groove.
```

Comment:
this changes logic. Current logic says:
> All of the following:
>   Varia Suit and Lay Any Bomb

Reply:
Whoops. I had forgotten about the return trip. So, yes it is a logic change, but in a positive way. The Gamma is no longer a PONR without Bombs/PBs, but I guess it can still be a PONR if you cant get enough height to leave. So, I guess it doesn't solve all issues with this check, but it does improve it which I prefer.

Reply:
worth noting that patching point of no returns != improving it.
AM2R is a really good example because me (and other people) nowadays actually prefer playing *with* point of no returns because it makes for slightly more interesting logic in that game.

In this situation it seems pretty indifferent to me tho. Theres a save station nearby so you can always try to peak the check

Reply:
Yes we should revisit because we are inconsistent across the board. Bomb Chamber we changed for example, but Spazer we did not. Plasma was changed too. Might be a good idea to revert all item rooms back to what they are in vanilla.

Reply:
DCR on stream: "hey miepee, can you please create an issue for this please? I keep forgetting that I want to change this"